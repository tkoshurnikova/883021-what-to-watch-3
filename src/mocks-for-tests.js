export const FilmDetails = {
  title: `Whispering Mist`,
  genre: `Comedy`,
  year: 1995,
  video: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

export const films = [
  {
    "title": `The No Flowers`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `Rings of Cloud`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `The Sword's Memory`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `The Voyagers of the Words`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `Shores in the Heart`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `Missing Illusion`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `The Unwilling Man`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  },
  {
    "title": `Teacher in the Hunter`,
    "image": `http://placehold.it/280x175`,
    "description": `qwerty`,
    "rating": 1.0,
    "numberOfVotes": 1982,
    "director": `Taika Di Caprio`,
    "actors": [`Ashley Cooper`, `Ramona Flowers`],
    "genre": `Dramas`,
    "year": 1900,
    "preview": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "video": `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    "background_color": `#ffffff`,
    "background_image": `some image`,
    "poster_image": `some image`
  }
];
