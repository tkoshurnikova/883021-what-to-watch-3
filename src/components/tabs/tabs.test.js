import React from "react";
import renderer from "react-test-renderer";
import Tabs from "./tabs.jsx";
import {films} from "../../mocks-for-tests.js";

const activeTab = `Overview`;

it(`Render Tabs`, () => {
  const tree = renderer
    .create(<Tabs
      film={films[0]}
      activeTab={activeTab}
      onTabClick={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
