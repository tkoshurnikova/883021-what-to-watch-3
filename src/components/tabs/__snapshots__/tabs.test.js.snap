// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Tabs 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      1.0
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Bad
      </span>
      <span
        className="movie-rating__count"
      >
        1982
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      qwerty
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Taika Di Caprio
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Ashley Cooper
         and other
      </strong>
    </p>
  </div>,
]
`;
